about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert-website</title>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+
+</head>
+
+<body>
+    <section class="top-nav">
+        <div></div>
+        <div class="top-nav-content">
+            <img src="./items/facebook-icon.png" alt="Facebook icon">
+            <img src="./items/twitter-icon.png" alt="twiter icon">
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
+    </section>
+    <header>
+        <a href="index.html" class="logo">
+            <img src="./items/Logo.png" alt="this is the image of the logo">
+        </a>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <div class="nav-lists">
+            <a href="./index.html" class="nav-link">Home</a>
+            <a href="" class="nav-link">Schedule</a>
+            <a href="" class="nav-link">Contact-Us</a>
+            <button>Get Ticket</button>
+
+        </div>
+    </header>
+    <section class="about-section">
+        <p class="about-section-firstParagraph">Hello!Music Fans</p>
+        <h1 class="about-heading">Muzik best for experience</h1>
+        <p class="about-section-secondParagraph">This is a platform for selling concert tickets and it is singer-three
+            to showcasing the ability of the global singers through providing the videos
+            of their lasts concerts and tour yes you can just go ahead and subscribe to the chanel for you to be updated.
+        </p>
+        <p class="about-third-paragraph">Please conact us per email for any futher questions on Muzik and Tickets</p>
+        <h2 class="about-email">concert@muzik.org</h2>
+        <hr>
+    </section>
+    <section class="tier-section">
+        <div class="tier-headding">
+            <h1> Grab your Tier here.! </h1>
```

Comment:
- [ ] This is a subtitle so it would be nice if you used a hx tag such as h2, h3 e.t.c instead of the `h1` tag which should be reserved fo the main title of the page, for semantic purposes